### PR TITLE
add ncl dewtemp_trh outputs

### PR DESCRIPTION
**Pull request by cyschneck**
Adds `dewtemp_trh` output values to use for `dewtemp` in applications (https://github.com/NCAR/geocat-applications/issues/100)

### CODE REVIEW

**Review by kafitzgerald — COMMENTED** [review]
It looks like it's fairly easy to programmatically generate and check inputs / outputs for this function.  Can this be thinned down a bit and live in GeoCAT applications?  I feel like that's what we did for some of the other functions.

Totally possible I missed something here too (apparently I did).

**Comment by anissa111** [comment]
Outside of the conversation on location that I'll leave to our slack conversation, I have a small suggestion. Assuming it would be minimal effort, I think it might be useful to include the corresponding inputs in the file as additional columns. As in, outputting columns of temp, rh, and then the calculated dewtemp. This would make it easier to recreate and compare over in geocat-applications, as well.

Also, the ncl script itself should live over in geocat-applications.

**Comment by cyschneck** [comment]
> Outside of the conversation on location that I'll leave to our slack conversation, I have a small suggestion. Assuming it would be minimal effort, I think it might be useful to include the corresponding inputs in the file as additional columns. As in, outputting columns of temp, rh, and then the calculated dewtemp. This would make it easier to recreate and compare over in geocat-applications, as well.
> 
> Also, the ncl script itself should live over in geocat-applications.

Removed `dewtemp_trh.ncl` and included the tk/rh as values in a comma separated columns with dewtemp

File: registry.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
-applications_files/README.md 1747f541b1f9fc81313fa2c3b3574df15f0ea910dc0874b709c6373cab935c02
+applications_files/README.md 170400b84f236f7d01fc0f64f3f6192404edd8a3dfc29647b0c69b7906b90741
+applications_files/ncl_outputs/dewtemp_trh.ncl 69f72f17d2a04aeccbef54343097c513cf2b7b9448e38a4a5e7d28713670bafa
+applications_files/ncl_outputs/dewtemp_trh_output.txt db95d50e76287382984bf40e1dcf2f79e9b8bbe72adcef0d110c59beceac8b11
```
Comment (kafitzgerald) [inline registry.txt:3]: I think these still need an update, but otherwise looks good to me.

**Review by kafitzgerald — CHANGES_REQUESTED** [review]
registry file needs an update, but otherwise looks good.

**Review by anissa111 — APPROVED** [review]

**Review by kafitzgerald — APPROVED** [review]

**Pull request merged**